server/portal/templates/includes/nav_cms.html

```diff
@@ -0,0 +1,12 @@
+{# @var className #}
+
+{# RFE: Use a custom element or React component whose code exists in one repo #}
+{# WARNING: Some markup is duplicated in other repositories #}
+{# SEE: https://confluence.tacc.utexas.edu/x/LoCnCQ #}
+<ul class="s-cms-nav  {{ className }}" id="cms-nav-pages" data-cms-nav-endpoint="/cms/nav/pages/markup/">
+
+  <!-- All top-level CMS controlled nav elements handled here. -->
+  {# FAQ: Content populated via JavaScript, below #}
+
+</ul>
+<script src="/static/site_cms/js/nav_cms.js"></script>
```

Comment:
__Explanation__: [The Javascript that was used (to populate CMS nav)](https://github.com/TACC/Frontera-Portal/pull/60/files#diff-b0d1bcca0e4b8af42b56dfcd3d05bf028cc5ba4ea89444ca2514ba0c293ea098L77) has [moved to the CMS](https://gitlab.tacc.utexas.edu/wma-cms/cms-site-template/blob/master/taccsite_cms/static/site_cms/js/nav_cms.js), so that both the Portal __and__ the [User Guide use identical Javascript to populate the CMS nav](https://bitbucket.org/taccaci/frontera-tech-docs/pull-requests/10#Lfrontera_theme/nav_cms.htmlT12). For now, the CMS will manage common scripting, until (perhaps) a shared repo exists for common scripts.